classes/03class/exercises/c03-docker02/nimmi89/README.md

```diff
@@ -0,0 +1,36 @@
+# C03-Docker02
+
+## Docker 
+- [Dockerfile.v1](Dockerfile.v1)
+- [Dockerfile.v2](Dockerfile.v2)
+
+## Command Execution Output
+- Curl command and its output of before changing the image:
+```
+$ curl http://localhost:8081
+<h1>DevOps Academy - Docker - Exercise c03-docker02</h1>
+$ curl http://localhost:8082
+<h1>DevOps Academy - Docker - Exercise c03-docker02</h1>
+```
+
+- Curl command and its output of after changing the image:
+```
+$ curl http://localhost:8081
+<h1>DevOps Academy - Docker - Exercise c03-docker02</h1>
+$ curl http://localhost:8082
+<h1>Changed !!!!DevOps Academy - Docker - Exercise c03-docker02</h1>
+```
+
+- Explain any difference between the responses of the webservers before and after changing the file locally:
+```
+>sudo docker run -itd -p 8081:80 exercise02:v1
```

Comment:
to avoid having to use sudo every time you can add your user to the docker group.